docs/application/dotnet/guides/user-interface/nui/ui-components.md

```diff
@@ -11,29 +11,33 @@ The following table lists the available UI components:
+| [RadioButton](./nui-components/RadioButton.md) | A RadioButton that can set selected or unselected status when user selects it. <br>[Guide ](./nui-components/RadioButton.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.RadioButton.html) | <img src="./screenshots/RadioButton.png" width="300"> <br>[C# code for this page](./codeExamples/RadioButton.cs) / [XAML page](./codeExamples/RadioButton.xaml) |
+| [ScrollableBase](./nui-components/ScrollableBase.md) | A ScrollableBase supports scrolling. <br>[Guide ](./nui-components/ScrollableBase.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.ScrollableBase.html)  | <img src="./screenshots/ScrollableBase.png" width="300"> <br>[C# code for this page](./codeExamples/ScrollableBase.cs) / [XAML page](./codeExamples/ScrollableBase.xaml) |
+| [Slider](./nui-components/Slider.md) | A slider that indicates a modifiable value within a specific range. <br>[Guide ](./nui-components/Slider.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.Slider.html)  | <img src="./screenshots/Slider.png" width="300"> <br>[C# code for this page](./codeExamples/Slider.cs) / [XAML page](./codeExamples/Slider.xaml) |
+| [Switch](./nui-components/Switch.md) | A switch that can be used as a selector. <br>[Guide ](./nui-components/Switch.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.Switch.html) | <img src="./screenshots/Switch.png" width="300"> <br>[C# code for this page](./codeExamples/Switch.cs) / [XAML page](./codeExamples/Switch.xaml) |
+| [TabBar](./nui-components/Tab.md)        | <div align="left">TabBar is a class which contains a set of TabButtons and has one of them selected.</div> <br>[Guide ](./nui-components/Tab.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.TabBar.html) |  <img src="./screenshots/TabButton.png" width="300"> <br>[C# code for this page](./codeExamples/Tab.cs) / [XAML page](./codeExamples/Tab.xaml) |
+| [TabButton](./nui-components/Tab.md)        | <div align="left">TabButton is a class which is used for selecting a content in the TabView.</div> <br>[Guide ](./nui-components/Tab.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.TabButton.html) |  <img src="./screenshots/TabButton.png" width="300"> <br>[C# code for this page](./codeExamples/Tab.cs) / [XAML page](./codeExamples/Tab.xaml) |
```

Comment:
Line 33: TabButton
The use of **< div >** tag is causing a line break between the description and the “Guide and API Documentation” hyperlinks. This is creating an inconsistency with the rest of the table format.